README.md

```
@@ -462,6 +808,39 @@ val response : String = try await openID4VP.shareVerifiablePresentation(vpTokenS
 
 This method will also notify the Verifier about the error by sending it to the response_uri endpoint over http post request. If response_uri is invalid and validation failed then Verifier won't be able to know about it.
 
+### constructErrorInfo
+
+- Receives an exception and constructs an error response map containing error code, message and optional state to be sent to the Verifier.
+
+```swift
+    let errorInfo: [String: Any] = openID4VP.constructErrorInfo(exception: Error)
+```
+
+###### Parameters
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)